source/_integrations/statistics.markdown

```diff
@@ -30,8 +30,10 @@ recorder:
 sensor:
   - platform: statistics
     entity_id: sensor.cpu
+    name: "CPU Statistics"
```

Comment:
These parameters are optional and thus should not be part of the example.
We try to keep examples as minimal as possible.

Nevertheless, thank you for willing to contribute 👍